src/controllers/auth.controller.js

```
@@ -43,24 +128,67 @@ const registerUser = async (req, res) => {
- * @description Đăng nhập người dùng đã đăng ký tài khoản, trả về access token và refresh token
- * Refresh token sẽ được cập nhật trên database, cũng như trả về cho client
- * Nếu phía front end có sử dụng dạng server dạng Backend For Frontend (BFF) thì có thể gửi token dưới dạng json cho phía Backend đó (hiện tại đang tính như vậy), nếu không thì gửi refresh_token dưới dạng một HttpOnly cookie 
- * @see https://owasp.org/www-community/HttpOnly
- * @see https://stackoverflow.com/questions/57650692/where-to-store-the-refresh-token-on-the-client
+ * @swagger
+ * /auth/sign-in:
+ *   post:
+ *     summary: Đăng nhập vào tài khoản của bạn
+ *     description: >
+ *       Đăng nhập vào hệ thống với email hoặc số điện thoại và mật khẩu của bạn. 
+ *       Sau khi đăng nhập thành công, bạn sẽ nhận được access token và refresh token. 
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)